websites/T/TruckersMP/presence.ts

```diff
@@ -1,380 +1,149 @@
 var presence = new Presence({
-  clientId: "624006279769227265" // CLIENT ID FOR YOUR PRESENCE
-});
-
-var item: any,
-  user: any,
-  item2: any,
-  item3: any,
-  server: any,
-  players: any,
-  output: any;
-
-var browsingStamp = Math.floor(Date.now() / 1000);
+    clientId: "821104573329440848"
+  }),
+  browsingStamp = Math.floor(Date.now() / 1000);
```

Comment:
its a recode and you still use var? and clientId doesnt need changing since the author is still active